sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py

```diff
@@ -26,6 +26,60 @@
 
 
 class Workspace(Resource):
+    """Azure ML workspace.
+
+    :param name: Name of the workspace.
+    :type name: str
+    :param description: Description of the workspace.
+    :type description: str
+    :param tags: Tags of the workspace.
+    :type tags: dict
+    :param display_name: Display name for the workspace. This is non-unique within the resource group.
+    :type display_name: str
+    :param location: The location to create the workspace in.
+        If not specified, the same location as the resource group will be used.
+    :type location: str
+    :param resource_group: Name of resource group to create the workspace in.
+    :type resource_group: str
+    :param hbi_workspace: Whether the customer data is of high business impact (HBI),
+        containing sensitive business information.
+        For more information, see
+        https://docs.microsoft.com/azure/machine-learning/concept-data-encryption#encryption-at-rest.
+    :type hbi_workspace: bool
+    :param storage_account: The resource ID of an existing storage account to use instead of creating a new one.
+    :type storage_account: str
+    :param container_registry: The resource ID of an existing container registry
+        to use instead of creating a new one.
+    :type container_registry: str
+    :param key_vault: The resource ID of an existing key vault to use instead of creating a new one.
+    :type key_vault: str
+    :param application_insights: The resource ID of an existing application insights
+        to use instead of creating a new one.
+    :type application_insights: str
+    :param customer_managed_key: Key vault details for encrypting data with customer-managed keys.
+        If not specified, Microsoft-managed keys will be used by default.
+    :type customer_managed_key: CustomerManagedKey
```

Comment:
Could you update the docstring types in this file to extended paths (i.e. ~azure.ai.ml.entities.CustomerManagedKey)?